### PR TITLE
remove branch for presubmit node e2e cri proxy

### PR DESCRIPTION
**Pull request by pacoxu**
follow up of https://github.com/kubernetes/test-infra/pull/33614 and https://github.com/kubernetes/test-infra/pull/33547

I compared this with `pull-kubernetes-node-e2e-resource-health-status`. The only diffs are 

```
+      branches:
+        # TODO(releng): Remove once repo default branch has been renamed
+        - master
+        - main
-    skip_branches:
-    - release-\d+\.\d+  # per-release image
```

node-e2e CIs that the current `/test` returns
```
/test pull-crio-cgroupv1-node-e2e-eviction
/test pull-crio-cgroupv1-node-e2e-features
/test pull-crio-cgroupv1-node-e2e-features-kubetest2
/test pull-crio-cgroupv1-node-e2e-hugepages
/test pull-crio-cgroupv1-node-e2e-resource-managers

/test pull-crio-cgroupv2-node-e2e-eviction
/test pull-crio-cgroupv2-node-e2e-hugepages
/test pull-crio-cgroupv2-node-e2e-resource-managers

/test pull-kubernetes-cos-cgroupv1-containerd-node-e2e
/test pull-kubernetes-cos-cgroupv1-containerd-node-e2e-features
/test pull-kubernetes-cos-cgroupv2-containerd-node-e2e
/test pull-kubernetes-cos-cgroupv2-containerd-node-e2e-eviction
/test pull-kubernetes-cos-cgroupv2-containerd-node-e2e-features
/test pull-kubernetes-cos-cgroupv2-containerd-node-e2e-serial

/test pull-kubernetes-node-e2e-alpha-ec2
/test pull-kubernetes-node-e2e-containerd-1-7-dra
/test pull-kubernetes-node-e2e-containerd-alpha-features
/test pull-kubernetes-node-e2e-containerd-ec2
/test pull-kubernetes-node-e2e-containerd-features
/test pull-kubernetes-node-e2e-containerd-features-kubetest2
/test pull-kubernetes-node-e2e-containerd-kubetest2
/test pull-kubernetes-node-e2e-containerd-serial-ec2
/test pull-kubernetes-node-e2e-containerd-serial-ec2-eks
/test pull-kubernetes-node-e2e-containerd-standalone-mode
/test pull-kubernetes-node-e2e-containerd-standalone-mode-all-alpha
/test pull-kubernetes-node-e2e-crio-cgrpv1-dra
/test pull-kubernetes-node-e2e-crio-cgrpv2-dra
/test pull-kubernetes-node-e2e-resource-health-status

/test all
pull-kubernetes-node-e2e-containerd

```

### CODE REVIEW

**Comment by pacoxu** [comment]
/cc @aojea @SergeyKanzhelev 
last try

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm
/approve

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 1]
```diff
@@ -4114,10 +4114,6 @@ presubmits:
                 memory: 10Gi
     - name: pull-kubernetes-node-e2e-cri-proxy-serial
       cluster: k8s-infra-prow-build
-      branches:
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:4117]: you should have branches or skip_branches to control which branches this runs on, related to forking the job per branch (different image versions and configuration on release branches)

this is not why your job isn't running.

Comment (pacoxu) [reply config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:4117]: fixed in https://github.com/kubernetes/test-infra/pull/33625

**Pull request merged**